gigascience/gigadb-website
# Fix outstanding issues in create account page

# Pull request for issue: #1542

This is a pull request for the following functionalities:

Handle remaining issues in http://gigadb.gigasciencejournal.com:9170/user/create

## How to test?

* Navigate to page
* Run the WAVE audit (chrome extension) and check that there are no "errors" reported
* Verify that the page looks ok
* a11y tests should pass:
```
cd playwright
npm i
npm test
```

## How have functionalities been implemented?

* Added contact email in the page, because captcha could give difficulties to some users and direct contact is the simplest fix here as an alternative to registration
* Removed "*" from screen readers because the inputs are already announced as required
* Added colors to checkbox if there is an error
![image](https://github.com/gigascience/gigadb-website/assets/143437854/79f65b3b-fb54-4d4f-8eab-2616c53fb1af)
* If there are errors, all of them appear on top
![image](https://github.com/gigascience/gigadb-website/assets/143437854/4804bf04-c59d-40f9-97b5-cc44fe9f231a)
* Changed error color for better contrast in light gray background

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
